templates/startup_script.sh.tmpl

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# create tamr user, to run tamr
+adduser --shell /bin/bash --home /home/tamr tamr
+sudo apt-get update
```

Comment:
This will fail the script on redhat based systems, which includes ec2 linux

Reply:
This isn't a requirement. I can just remove it.